demisto_sdk/commands/test_content/TestContentClasses.py

```diff
@@ -1263,8 +1263,28 @@ def _set_integration_params(
             self._change_placeholders_to_values(server_url, conf)
             for conf in integration_params
         ]
-
-        if integration_params:
+        if self.name == "Core REST API":
+            if IS_XSIAM:
+                self.configuration.params = {  # type: ignore
+                    "url": server_url,
+                    "creds_apikey": {
+                        "identifier": str(self.build_context.auth_id),
+                        "password": self.build_context.api_key,
+                    },
+                    "auth_method": "Standard",
+                    "insecure": True,
+                }
+            else:
+                self.configuration.params = {  # type: ignore
+                    "url": "https://localhost",
+                    "creds_apikey": {
+                        "identifier": "",
+                        "password": self.build_context.api_key,
+                    },
+                    "auth_method": "Standard",
+                    "insecure": True,
+                }
```

Comment:
we can avoid code duplication, e.g. `"url": server_url if IS_XSIAM else "https://localhost"` etc